misc/io.github.Tremulous.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Daniel Rusek <mail@asciiwolf.com> -->
+<component type="desktop-application">
+  <id>com.grangerhub.Tremulous</id>
+  <project_license>GPL-2.0</project_license>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <provides>
+    <id>tremulous.desktop</id>
+  </provides>
+  <launchable type="desktop-id">com.grangerhub.Tremulous.desktop</launchable>
```

Comment:
The launchable needs to match the desktop file name. So either rename the desktop file name or change it to the actual desktop file name.